dataactvalidator/config/sqlrules/fabs37_detached_award_financial_assistance_3.sql

```diff
@@ -0,0 +1,24 @@
+-- For new assistance awards (ActionType = A), the CFDA_Number must have been registered
```

Comment:
This is not accurate, you're checking if the CFDA Number has been registered period, not based on an action type. This comment should reflect the rule that is being printed to users.